Casks/m/macsymbolicator.rb

```diff
@@ -10,6 +10,7 @@
   depends_on macos: ">= :mojave"
 
   app "MacSymbolicator.app"
+  binary "#{appdir}/MacSymbolicator.app/Contents/MacOS/MacSymbolicatorCLI", target: "macsymbolicator-cli"
```

Comment:
```suggestion
  binary "#{appdir}/MacSymbolicator.app/Contents/MacOS/MacSymbolicatorCLI", target: "mac-symbolicator-cli"
```

This matches the upstream documentation.